### ISSUE TITLE
formatDateTime not allowed with extra locale settings

### ISSUE THREAD

**Issue by cxbuilders** [microsoft/botbuilder-dotnet#5022]
In the [adaptive expressions documentation](https://docs.microsoft.com/en-us/azure/bot-service/adaptive-expressions/adaptive-expressions-prebuilt-functions?view=azure-bot-service-4.0#formatDateTime) it shows that formatDateTime can accept an additional locale parameter:
![image](https://user-images.githubusercontent.com/49873947/100988865-787c4d00-3550-11eb-8786-1a8d8a1b36ca.png)

The validation in Composer however does not allow that:
![image](https://user-images.githubusercontent.com/49873947/100989157-d01ab880-3550-11eb-8144-541da53be529.png)

**Comment by cosmicshuai** [comment]
I know the reason why this happened. There is gap between the implementation of locale support between javascript and dotnet. I will add locale support for datetime functions in botbuulder-js.

**Comment by taicchoumsft** [comment]
@cxbuilders , are you using the dotnet sdk and which version are you on? We will try to replicate this bug.

**Comment by anusharavindrar** [comment]
Looking into this. @cxbuilders, can you respond to @taicchoumsft's query?

**Comment by cxbuilders** [comment]
@anusharavindrar .net it is and version 1.2.2

**Comment by anusharavindrar** [comment]
@cxbuilders, thanks for the above info. Let me try to repro this and update the thread.

**Comment by gabog** [comment]
Hi @anusharavindrar, can you provide an update on this one?

Thanks

**Comment by anusharavindrar** [comment]
I am able to reproduce the bug on Composer v1.3.0 
![image](https://user-images.githubusercontent.com/48731427/103687859-cc11fa00-4f45-11eb-8e9a-64c89c641ca4.png)


@cosmicshuai, do you think this might be related to your above comment?

> I know the reason why this happened. There is gap between the implementation of locale support between javascript and dotnet. I will add locale support for datetime functions in botbuilder-js.

**Comment by cosmicshuai** [comment]
@ahmedaashour Right, the validation in Composer depends on the Adaptive-Expression and LG js package. 
I have a PR for solving this issue. [PR for locale](https://github.com/microsoft/botbuilder-js/pull/3162)
After this PR merged, I will update the dependancies of LG and Expressions.

**Comment by LeeParrishMSFT** [comment]
@cosmicshuai I see the PR is now merged, when do you think you'll have the LG and Expressions updated so we can close this? Thanks!
